### DISCUSSION TITLE
how can use softethervpn just for browsers ?

### DISCUSSION THREAD

**Discussion by rerush635** [SoftEtherVPN/SoftEtherVPN#1688] [General]
hello 

Anyone know how can to use softethervpn just for browsers ? something like split tunnel 

or we have something for config SSTP free servers on browsers like chrome ? 

thanks for answer

**Comment by shipitsin-ilia** [comment]
for browser should work http proxy, not vpn.


alternatively, browser addon which emulates vpn inside browser, but I haven't heard of such plugins for SE VPN
